ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java

```diff
@@ -2,18 +2,26 @@
 
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.services.MemberIngestService;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.validation.LdesShaclValidator;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class LdesMemberIngestionController {
 
-	private final MemberIngestService memberIngestService;
+	@Autowired
+	MemberIngestService memberIngestService;
+	@Autowired
+	private LdesShaclValidator validator;
 
-	public LdesMemberIngestionController(final MemberIngestService memberIngestService) {
-		this.memberIngestService = memberIngestService;
+	@InitBinder
```

Comment:
Oei waarom is dit nodig?

Reply:
To use  @Validated in the rest controller, this validator needs to be binded. therefor the initbind